spec/services/payments/lose_dispute_service_spec.rb

```diff
@@ -24,7 +24,8 @@
     end
 
     context "when payable is not found" do
-      let(:payment) { create(:payment, payable: create(:payment_request)) }
+      let(:payment_request) { create(:payment_request) }
+      let(:payment) { create(:payment, payable: payment_request) }
```

Comment:
I don't understand the difference.

Reply:
No difference actually. It's probably related to an other change I've made and finally roll-backed. I will restore it to the previous state